extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/internal/LifxLightCommunicationHandler.java

```diff
@@ -164,7 +164,7 @@ public void start() {
             broadcastPacket(packet);
 
         } catch (Exception ex) {
-            logger.error("Error occured while initializing LIFX handler: " + ex.getMessage(), ex);
+            logger.error("Error occurred while initializing LIFX handler: " + ex.getMessage(), ex);
```

Comment:
As you already touched this logging statement, can you fix it and stop using string building (use parameters instead, so the logging framework will build the string only if logging is done).